<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminVehicles</class>
 <widget class="QWidget" name="AdminVehicles">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1160</width>
    <height>698</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
font-family:century gothic;
font-size:17px;
background:#333;

}


QPushButton
{
background:blue;
border-radius:15px;
}
QToolButton
{
background:blue;
border-radius:60px;
}
QTextEdit
{
bakcground:#333;
border-radius:60px;
}
QLabel
{
color:white;
}
QWidget
{
color:white;
}
QPushButton:hover
{
background:grey;
}
QLineEdit
{
background:transparent;
border:none;
color:white;
border-bottom:1px solid #717072;
}
QTableView
{
background:#333;
}
QHeaderView {
background:#333;
}
QTextBrowser {
font-family:century gothic;
font-size:30px;
}&lt;
</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1989</width>
     <height>701</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Main</string>
    </attribute>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>520</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_recherche">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>520</y>
       <width>140</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>380</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Statistics</string>
     </property>
    </widget>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>30</y>
       <width>821</width>
       <height>441</height>
      </rect>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="triID">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>520</y>
       <width>81</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ID_desc">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>550</y>
       <width>81</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Brand</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>480</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Order</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>580</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>PDF</string>
     </property>
    </widget>
    <widget class="QPushButton" name="moisprec">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>440</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Previous</string>
     </property>
    </widget>
    <widget class="QPushButton" name="moisuivant">
     <property name="geometry">
      <rect>
       <x>980</x>
       <y>440</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::WheelFocus</enum>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="text">
      <string>Next</string>
     </property>
     <property name="autoRepeatDelay">
      <number>296</number>
     </property>
     <property name="autoRepeatInterval">
      <number>91</number>
     </property>
    </widget>
    <widget class="QPushButton" name="triCat">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>580</y>
       <width>81</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>VIN</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_11">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>470</y>
       <width>221</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Calculate After Taxes</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>102</x>
       <y>430</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>520</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>430</y>
       <width>56</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Price</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>520</y>
       <width>56</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Total</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>490</y>
       <width>221</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Search For A Certain Car</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>258</width>
       <height>291</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>License Plate</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_id">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Brand</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_centre"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>DPMC</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="dateEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Category</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_centre_2"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>VIN</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_piste"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>CLIENT ID</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_cin"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>320</y>
       <width>221</width>
       <height>54</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>Insert</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_8">
          <property name="text">
           <string>Display</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>Update</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_3">
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
